source/common/router/header_formatter.cc

```diff
@@ -126,6 +126,12 @@ RequestInfoHeaderFormatter::RequestInfoHeaderFormatter(absl::string_view field_n
       return RequestInfo::Utility::formatDownstreamAddressNoPort(
           *request_info.downstreamRemoteAddress());
     };
+  } else if (field_name == "ORIGINAL_DST" ||
```

Comment:
Do we need "ORIGINAL_DST"? Would prefer to only have one and the other is more common with the new layout.